src/zodbupdate/tests.py

```diff
@@ -1266,11 +1293,76 @@ def test_loaded_renames_override_missing_interfaces(self):
         self.assertEqual({}, renames)
 
 
-def test_suite():
-    suite = unittest.TestSuite()
-    suite.addTest(unittest.makeSuite(TestsBasics))
-    if six.PY2:
-        suite.addTest(unittest.makeSuite(Python2Tests))
-    if six.PY3:
-        suite.addTest(unittest.makeSuite(Python3Tests))
-    return suite
+class RelStorageHFMixin(object):
+    """
+    Mixin to create a history-free RelStorage using SQLite.
+    """
+    def _makeStorage(self):
+        from ZODB import config
+        return config.storageFromString(
+            """
+            %%import relstorage
+            <relstorage>
+                keep-history false
+                blob-dir %s
+                <sqlite3>
+                   data-dir %s
+                </sqlite3>
+            </relstorage>
+            """ % (os.path.join(self.tmpblob, 'blobs'), self.tmpblob,)
+        )
+
+    _tearDownStorage = lambda self: None
```

Comment:
This is a bit surprising to me.

If _tearDownStorage is meant to remove all the data files, then this one doesn't do that.

If _tearDownStorage is meant to remove any additional data files that live outside the temporary directory that gets shutils.rmtree'd, then why does the base class take pains to remove the file storage files manually?  It could be a `pass` and relstorage wouldn't need to override it.

Reply:
Yeah, that's poorly factored. In the effort to keep changes minimal, I left an unclear division of responsibility. I'll clean that up. 

Thanks for calling that out!

Reply:
> If _tearDownStorage is meant to remove any additional data files that live outside the temporary directory that gets shutils.rmtree'd, then why does the base class take pains to remove the file storage files manually?

The underlying reason is that originally, the FileStorage data file and its blobs directory were completely different parts of the filesystem tree. Only the blobs directory was `shutils.rmtree`d. I changed that to put both the data file and blobs directory under the temporary directory (making FileStorage lay things out like RelStorage was).

Reply:
Oh, oops, and I didn't even notice!

I like your changes a lot!